etna/metrics/functional_metrics.py

```diff
@@ -41,6 +41,45 @@ def _get_axis_by_multioutput(multioutput: str) -> Optional[int]:
         assert_never(multioutput_enum)
 
 
+def mse_with_missing_handling(y_true: ArrayLike, y_pred: ArrayLike, multioutput: str = "joint") -> ArrayLike:
```

Comment:
We should discuss naming and the fact that sklearn's `mse` is in our `etna/metrics/__init__.py`.

Reply:
We can name it `mean_squared_error`. Mentioning missing handling in the name seems a bit excessive. Better to leave this clarification to the documentation.

Reply:
If we have both `mse` from sklearn and our `mean_squared_error` it could be confusing.

We could remove sklearn's version, we haven't mentioned it in our public documentation (but have it in `etna/metrics/__init__.py`).

I'm also a bit of worried that by replacing sklearn's mse with our mse we changed list of available kwargs.